src/components/Search/SearchBar.js

```diff
@@ -0,0 +1,171 @@
+// import { ReactComponent as MagnifyingGlass } from '../assets/search.svg';
+
+
+class SearchBar extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            fileName: '',
+            countyName: '',
+            children: []
+        }
+    }
+
+    componentDidMount() {
+        const children = [];
+        const { Option } = Select;
+
+        for (let county of COUNTIES) {
+            const child = {
+                key: `${county.geoid}-county`,
+                button: []
+            } 
+            child.button.push(
+                <Option
+                    key={`${county.geoid}-county`}
+                    value={county.geoid}>
+                    {county.name}, {county.usps}
+                </Option>
+            )
+            children.push(child);
+        }
+
+        this.setState({children})
+    }
+
+    handleCountySelect = (event) => {
+
+        const item = COUNTIES.filter(county => county.geoid === event)[0];
+
+        this.props.onCountySelect(item);
+        this.setState({
+            countyName: `${item.name}, ${item.usps}`
+        })
+
+        // use for when files are on public internet
+        // axios.get(event.path)
+        //     .catch(error => {
+        //         console.log('error', error);
+        //     })
+        //     .then(response => {
+        //         // console.log('event', event)
+        //         // console.log('response', response)
+        //         const json = response.data;
+        //         this.props.onCountySelect(json);
+        //         this.setState({
+        //             'countyName': event.name + ', ' + event.usps
+        //         })
+        //     });
+    }
+
+    handleUpload = (info) => {
+        // antd uploader doesn't provide data to fileRead
+        // if (info.file.status !== 'uploading') {
+        //     console.log(info.file, info.fileList);
+        //   }
+        //   if (info.file.status === 'done') {
+        //     message.success(`${info.file.name} file uploaded successfully`);
+        //   } else if (info.file.status === 'error') {
+        //     message.error(`${info.file.name} file upload failed.`);
+        //   }
+        
+        // event sent from old bootstrap upload element
+        // if (event.target.files.length > 0) {
+        //     const file = event.target.files[0]
+        //     if (this.validateSize(file) && this.validateType(file)) {
+        //         const fileReader = new FileReader();
+        //         fileReader.onload = () => {
+        //             const json = JSON.parse(fileReader.result);
+        //             this.props.onFileUpload(json);
+        //             this.setState({'fileName': file.name})
+        //         }
+        //         fileReader.readAsText(file)
+        //     }
+        // }
+        console.log('upload')
+    }
+
+    validateSize = (file) => {
+        let size = 100000000;  //100mb
```

Comment:
use `1024 x 1024 x 100`
 
1024 bytes = 1kb
1024 bytes  x 1024 bytes = 1mb

Reply:
Another way to make this code and user friendly is to store the mb limit as it's own const and use it in the message

```
    validateSize = (file) => {
        let size = 100;  //100mb
        let error = '';
        if (file.size > (size * 1024 * 1024)) {
            error = `File is too large, please upload a file less than ${size}mb`;
            // react-toastify beautifies notifications
            alert(error);
        } else {
            return true;
        }
    }
```